langgraph_supervisor/supervisor.py

```diff
@@ -255,9 +255,11 @@ def web_search(query: str) -> str:
 
         agent_names.add(agent.name)
 
-    handoff_destinations = _get_handoff_destinations(tools or [])
-    if handoff_destinations:
-        if missing_handoff_destinations := set(agent_names) - set(handoff_destinations):
+    extracted_handoff_destinations = _get_handoff_destinations(
```

Comment:
why change?

Reply:
`_get_handoff_destinations` returns a `list[str]`, but then down in the else block, `handoff_destinations` becomes a `list[BaseTool]`. This change is to avoid having to cast the `handoff_destinations` variable from `list[str]` to `list[BaseTool]` later on.